src/uk/gov/hmcts/pipeline/AKSSubscriptions.groovy

```diff
@@ -26,5 +27,8 @@ class AKSSubscriptions {
     def prodName = steps.env.AKS_PROD_SUBSCRIPTION_NAME ?: 'DCD-CFTAPPS-PROD'
     def prodKvName = steps.env.PROD_AKS_KEY_VAULT ?: 'cft-apps-prod'
     prod = new AKSSubscription(steps, prodName, prodKvName, 'prod', true)
+
+    def demoName = steps.env.AKS_DEMO_SUBSCRIPTION_NAME ?: 'DCD-CFTAPPS-DEMO'
+    demo = new AKSSubscription(steps, demoName, 'cftapps-demo', 'demo', true)
```

Comment:
Do it same as prod ? and get it from env variable? like https://github.com/hmcts/cnp-jenkins-library/pull/596/files#diff-6e016a19ecdb9ae05ca73906b312360cR29

Reply:
never mind, ignore it.